lib/app/components/pages/home/views/home_view.dart

```diff
@@ -70,37 +84,39 @@ class HomeView extends GetView<HomeController> {
                           },
                           separatorBuilder: (context, index) =>
                               const SizedBox(height: 10),
-                        )
-                      : Column(
-                          children: const [
-                            SizedBox(height: 30),
-                            Center(
-                              child: Text(
-                                '検索結果がありません',
-                                style: TextStyle(fontSize: 18),
-                              ),
+                        ))
+                    : Column(
+                        children: const [
+                          SizedBox(height: 30),
+                          Center(
+                            child: Text(
+                              '検索結果がありません',
+                              style: TextStyle(fontSize: 18),
                             ),
-                          ],
-                        ),
-                ),
+                          ),
+                        ],
+                      ),
               ),
             ),
           ),
-          Center(
-            child: ElevatedButton(
-              style: ElevatedButton.styleFrom(
-                fixedSize: Size(MediaQuery.of(context).size.width * 0.3, 25),
-                backgroundColor: const Color.fromRGBO(236, 188, 179, 1),
-                foregroundColor: Colors.white,
-                shape: const StadiumBorder(),
-                elevation: 0,
-              ),
-              onPressed: () {
-                // todo トップに戻る
-              },
-              child: const Text('TOPに戻る'),
-            ),
-          )
+          Obx(() => Visibility(
+              visible: controller.needScrollToTop.value,
+              child: Center(
+                child: ElevatedButton(
+                  style: ElevatedButton.styleFrom(
+                    fixedSize:
+                        Size(MediaQuery.of(context).size.width * 0.3, 25),
+                    backgroundColor: const Color.fromRGBO(236, 188, 179, 1),
+                    foregroundColor: Colors.white,
+                    shape: const StadiumBorder(),
+                    elevation: 0,
+                  ),
+                  onPressed: () {
+                    controller.scrollController.jumpTo(0);
+                  },
+                  child: const Text('TOPに戻る'),
+                ),
+              )))
```

Comment:
```suggestion
              ),
             ),
           )
```
細かい指摘で申し訳ないですが、コンマ入れて改行して欲しいです。
VSCodeの設定でコンマ入れると改行されると思います。
[Code formatting](https://docs.flutter.dev/development/tools/formatting#using-trailing-commas)
上記のリンクにもあるんですが、`flutter format`コマンドでも修正できるみたいなので、
適宜改行してみやすくしてほしいです🙏

Reply:
ご指摘ありがとうございます。修正いたしました。